usr/mgmt_ipc.h

```diff
@@ -92,9 +92,11 @@ typedef struct iscsiadm_rsp {
 						ISCSI_STATS_CUSTOM_MAX)
 		struct ipc_msg_getstats {
 			struct iscsi_uevent ev;
-			struct iscsi_stats stats;
 			char custom[sizeof(struct iscsi_stats_custom) *
 					ISCSI_STATS_CUSTOM_MAX];
+      /* field 'stats' with variable sized type 'struct iscsi_stats' 
+       * must be at the end of a struct or class is a GNU extension */
+      struct iscsi_stats stats;                            
```

Comment:
Also indented incorrectly. And why did this one get moved, when it doesn't seem to be part of the structures you claim are variable-sized?